central/networkgraph/flow/datastore/internal/store/postgres/store.go

```diff
@@ -310,14 +320,14 @@ func (s *flowStoreImpl) UpsertFlows(ctx context.Context, flows []*storage.Networ
 	})
 }
 
-func (s *flowStoreImpl) retryableUpsertFlows(ctx context.Context, flows []*storage.NetworkFlow, _ timestamp.MicroTS) error {
+func (s *flowStoreImpl) retryableUpsertFlows(ctx context.Context, flows []*storage.NetworkFlow, lastUpdateTS timestamp.MicroTS) error {
```

Comment:
Why it was ignored before?
I had an idea to auto populate this field on upsert just before passing data to DB but it looks like everything was ready to accept this value

Reply:
I honestly don't know. Maybe at some point we used it to set `LastSeen` but your guess is as good as mine 😅

Reply:
That probably would have been something I did.  But since it was 3 years ago, I cannot tell you why